File: aspnet-core/src/MyABP.Core/Operation/Article.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyABP.Operation
{
    public class Article: AuditedEntity<int>
    {
        [MaxLength(50)]
        public string Title { set; get; }

        [MaxLength(200)]
        public string Img { set; get; }

        public int Sort { set; get; }

        public string Content { set; get; }

        [Required]
        public int Type { set; get; }

        [Required]
        public int Status { set; get; }

    }
}

File: vue/src/store/entities/article.ts
import  Entity from './entity'

export default  class article extends Entity<number> {
    title:string;
    img:string;
    sort:number;
    type:number;
    status:number;
    content:string;

    constructor() {
        super();
        this.sort = 0;
        this.type = 1;
        this.status = 0;
        this.content = '';
    }
}